scripts/!mods_preload/dynamic_perks.nut

```diff
@@ -25,10 +25,8 @@
 	::Hooks.registerJS("ui/mods/mod_dynamic_perks/generic_perkgroups_module.js");
 	::Hooks.registerCSS("ui/mods/mod_dynamic_perks/generic_perkgroups_module.css");
 
-	foreach(file in this.IO.enumerateFiles("ui/mods/mod_dynamic_perks/hooks"))
-	{
-		::Hooks.registerJS(file + ".js");
-	}
+	::Hooks.registerJS("ui/mods/mod_dynamic_perks/hooks/screens/character/modules/character_screen_right_panel/character_screen_perks_module.js");
+	::Hooks.registerCSS("ui/mods/mod_dynamic_perks/hooks/screens/character/modules/character_screen_right_panel/character_screen_perks_module.css");
 });
```

Comment:
It'd make more sense to me to fix the loop so it handles both .css and .js files but obviously not a dealbreaker

Reply:
Can be done later on if there's more files